src/scripts/create_arc.py

```diff
@@ -72,6 +80,72 @@ def main() -> None:
             & test_df["option_d"].notnull()
         ]
 
+        # Remove the samples with overly short or long texts
+        train_df = train_df[
+            (train_df.instruction.str.len() >= MIN_NUM_CHARS_IN_INSTRUCTION)
+            & (train_df.instruction.str.len() <= MAX_NUM_CHARS_IN_INSTRUCTION)
+            & (train_df.option_a.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (train_df.option_a.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+            & (train_df.option_b.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (train_df.option_b.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+            & (train_df.option_c.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (train_df.option_c.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+            & (train_df.option_d.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (train_df.option_d.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+        ]
+        val_df = val_df[
+            (val_df.instruction.str.len() >= MIN_NUM_CHARS_IN_INSTRUCTION)
+            & (val_df.instruction.str.len() <= MAX_NUM_CHARS_IN_INSTRUCTION)
+            & (val_df.option_a.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (val_df.option_a.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+            & (val_df.option_b.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (val_df.option_b.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+            & (val_df.option_c.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (val_df.option_c.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+            & (val_df.option_d.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (val_df.option_d.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+        ]
+        test_df = test_df[
+            (test_df.instruction.str.len() >= MIN_NUM_CHARS_IN_INSTRUCTION)
+            & (test_df.instruction.str.len() <= MAX_NUM_CHARS_IN_INSTRUCTION)
+            & (test_df.option_a.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (test_df.option_a.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+            & (test_df.option_b.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (test_df.option_b.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+            & (test_df.option_c.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (test_df.option_c.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+            & (test_df.option_d.str.len() >= MIN_NUM_CHARS_IN_OPTION)
+            & (test_df.option_d.str.len() <= MAX_NUM_CHARS_IN_OPTION)
+        ]
+
+        def is_repetitive(text: str) -> bool:
+            """Return True if the text is repetitive."""
+            max_repetitions = max(Counter(text.split()).values())
+            return max_repetitions > 50
```

Comment:
Why 50? Shouldn't this at least be a constant MAX_NUM_REPETITIONS (or an argument, of course)?